impeller/renderer/backend/metal/compute_pass_mtl.mm

```diff
@@ -248,17 +248,29 @@ static bool Bind(ComputePassBindingsCache& pass,
     // https://github.com/flutter/flutter/issues/110619
 
     // For now, check that the sizes are uniform.
-    FML_DCHECK(grid_size == thread_group_size);
+    // FML_DCHECK(grid_size == thread_group_size);
```

Comment:
I think we should rethink this API. What would make sense to me is 1. specifying the units of work in a grid size.

Maybe there are cases where we need to control threadgroup size? not sure